wagtail/admin/localization.py

```diff
@@ -109,10 +109,38 @@ def get_js_translation_strings():
         'WEEKDAYS_SHORT': [str(WEEKDAYS_ABBR[d % 7]) for d in range(-1, 6)],
 
         # used by bulk actions
-        'NUM_PAGES_SELECTED_SINGULAR': _('1 page selected'),
-        'NUM_PAGES_SELECTED_PLURAL': _("{0} pages selected"),
-        'NUM_PAGES_SELECTED_ALL': _("All {0} pages on this screen selected"),
-        'NUM_PAGES_SELECTED_ALL_IN_LISTING': _("All pages in listing selected"),
+        'BULK_ACTIONS': {
+            'page': {
```

Comment:
We haven't nested `STRINGS` before, but I think that's okay here (the alternative would be to store this information on the template). However, I think the change in case is confusing: either `BULK_ACTIONS` and `page` (for instance) should both be capitalised, or neither should, as they both do a similar thing here, exposing another strings object.

Reply:
Okay, I'll change them to all caps